### DISCUSSION TITLE
HTTPS redirection Weblate 3.9.1

### DISCUSSION THREAD

**Discussion by phandja** [WeblateOrg/weblate#7276] [General]
We are facing some issues with the https redirection on Weblate 3.9.1.
Our settings are bellow, as required in the documentation:

`ENABLE_HTTPS` = True
`SECURE_SSL_REDIRECT` = ENABLE_HTTPS
`SECURE_HSTS_SECONDS` = 1
`SECURE_HSTS_INCLUDE_SUBDOMAINS` = True
`SECURE_HSTS_PRELOAD` = True
`SESSION_COOKIE_SECURE` = True

I have tried for days but the redirection still not working. Is something wrong in my settings? I am using Docker.

**Comment by nijel** [comment]
Most likely the https status is not correctly passed to the wsgi process. See https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-SECURE_PROXY_SSL_HEADER
